whatsapp.py

```diff
@@ -97,6 +102,10 @@ def main(path, name):
     """Create and generate a word cloud image:"""
     ax.get_legend().remove()
     for author in Message.authors:
+        # revert name
+        if author[::-1] in df["author"].unique():
```

Comment:
I'm confused, why do you need to do this here if you've already done it on row 68?

Reply:
it doesn't change the class, only the df. so when you take the authors from Message.authors it's still in the old order.
I thought it best to not convert it in the class, since when you print the dfs it does know how to read it right.

Reply:
but that way you are doing it twice - once in the df, and once in the class. if we can take care of it in the class, why not?

Reply:
the process takes the string and reverts it (abc -> cba). while you print dfs, the hebrew works fine, but while printing plots it reverts. 
I don't want to revert the class so if we'll add more dfs after the plots it won't be backwords. maybe we need to consider the pipeline for that (create two dfs, one reversed and one not?)

Reply:
Oh, I see now. So we might need to come up with a solution that reverts specifically in plots, or something of the sort.  Lets open an issue for that and think about a pipline, like you said